docs/modules/ROOT/pages/contributor-guide/release-guide.adoc

```diff
@@ -190,79 +201,54 @@ For instance, one could search mail with subject:
 
 == Update the Apache Camel Quarkus distribution subversion repository
 
-Once the staging repository has been released, the Apache Camel Quarkus distribution subversion repository should be updated as sketched below.
+Once the staging repository has been released, the Apache Camel Quarkus distribution subversion repository should be updated.
+
+In a first step, one would simply need to promote the deliverables from the development to the distribution svn repository by invoking the `promote-release.sh`.
 
-If you are doing this for the first time you need to checkout the Apache Camel Quarkus distribution subversion repository:
 [source,shell]
 ----
-$ svn checkout 'https://dist.apache.org/repos/dist/release/camel' camel-releases-dist
+$ export VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout | sed 's|-SNAPSHOT||') # the version you are releasing, e.g. 0.1.0
+$ cd release-utils/scripts/
+$ ./promote-release.sh $VERSION $VERSION
 ----
 
-In case you have performed the above step during some release in the past, you need to update your working copy:
+In a second step, it is needed to check that useless versions are deleted from the svn repository.
```

Comment:
@oscerd Trying to make sense of the release scripts in the procedure, Is it really mandated to delete old deprecated versions from the distribution svn repository ?

Reply:
Yes, it's still true. But not really important.